depends/patches/miniupnpc/dont_use_wingen.patch

```diff
@@ -0,0 +1,26 @@
+commit e8077044df239bcf0d9e9980b0e1afb9f1f5c446
+Author: fanquake <fanquake@gmail.com>
+Date:   Tue Aug 18 20:50:19 2020 +0800
+
+    Don't use wingenminiupnpcstrings when generating miniupnpcstrings.h
+
+    The wingenminiupnpcstrings tool is used on Windows to generate version
+    information. This information is irrelevant for us, and trying to use
+    wingenminiupnpcstrings would cause builds to fail, so just don't use it.
+
+    We should be able to drop this once we are using 2.1 or later. See
+    upstream commit: 9663c55c61408fdcc39a82987d2243f816b22932.
+
+diff --git a/Makefile.mingw b/Makefile.mingw
+index 574720e..fcc17bb 100644
+--- a/Makefile.mingw
++++ b/Makefile.mingw
+@@ -74,7 +74,7 @@ wingenminiupnpcstrings:	wingenminiupnpcstrings.o
+
+ wingenminiupnpcstrings.o:	wingenminiupnpcstrings.c
+
+-miniupnpcstrings.h: miniupnpcstrings.h.in wingenminiupnpcstrings
++miniupnpcstrings.h: miniupnpcstrings.h.in
+ 	wingenminiupnpcstrings $< $@
```

Comment:
Probably, removing `miniupnpcstrings.h` target completely is more correct patch.

Reply:
I'd rather keep these as the equivalents of the commands they are replacing.

Reply:
```diff
--- Makefile.mingw.OLD	2016-01-19 12:26:58.000000000 +0200
+++ Makefile.mingw.NEW	2020-08-25 11:03:57.144468741 +0300
@@ -70,13 +70,6 @@
 listdevices: listdevices.o libminiupnpc.a
 	$(CC) -o $@ $^ $(LDLIBS)
 
-wingenminiupnpcstrings:	wingenminiupnpcstrings.o
-
-wingenminiupnpcstrings.o:	wingenminiupnpcstrings.c
-
-miniupnpcstrings.h: miniupnpcstrings.h.in wingenminiupnpcstrings
-	wingenminiupnpcstrings $< $@
-
 minixml.o:	minixml.c minixml.h
 
 upnpc.o:	miniwget.h minisoap.h miniupnpc.h igd_desc_parse.h
```

Reply:
> I'd rather keep these as the equivalents of the commands they are replacing.

ok

Reply:
Agree with both of these comments :)